api/remote.go

```diff
@@ -0,0 +1,52 @@
+package api
+
+import (
+	"io/ioutil"
+	"net/http"
+
+	util "github.com/datatogether/api/apiutil"
+	"github.com/qri-io/qri/lib"
+	"github.com/qri-io/qri/p2p"
+)
+
+// RemoteHandlers wraps a request struct to interface with http.HandlerFunc
+type RemoteHandlers struct {
+	lib.RemoteRequests
+}
+
+// NewRemoteHandlers allocates a RemoteHandlers pointer
+func NewRemoteHandlers(node *p2p.QriNode) *RemoteHandlers {
+	req := lib.NewRemoteRequests(node, nil)
+	return &RemoteHandlers{*req}
+}
+
+// ReceiveHandler is the endpoint for remotes to receive daginfo
+func (h *RemoteHandlers) ReceiveHandler(w http.ResponseWriter, r *http.Request) {
+	switch r.Method {
+	case "POST":
```

Comment:
most other handlers include an `OPTIONS` select case for CORS requests that just sends back an empty ok handler, I think it's actually the right decision to _not_ have that here, just wanted to point out why it exists elsewhere in case we end up doing publish-to-remote from a brower.